### PR TITLE
refactor: add filter keyword function to replace spaces with %20

### LINKED ISSUE #119: [BUG]: Server crashes due to twitter searchs
### What bug did you encounter?

After installing the dependencies and start the server, it crashes. After looking through the console, I found that there are some issues with the twitter search. 
I wonder why add the `keywords` on the search query, I think that caused the issue :
```js
 await axios.get(
      'https://twitter.com/search?p=from%3A' +
        twitterAccount.account +
        `${keyword}`
    );
```
The simplified error is here: 
```sh
  address: '104.244.42.1',
  port: 443,
    headers: {
      Accept: 'application/json, text/plain, */*',
      'User-Agent': 'axios/0.26.1'
    },
    method: 'get',
    url: 'https://twitter.com/search?p=from%3ABarackObamapetrol prices',
    data: undefined
  },
  request: <ref *4> Writable {
      protocol: 'https:',
      path: '/search?p=from%3ABarackObamapetrol%20prices',
      method: 'GET',
      headers: {
        Accept: 'application/json, text/plain, */*',
        'User-Agent': 'axios/0.26.1'
      },
      agent: undefined,
      agents: { http: undefined, https: undefined },
      auth: undefined,
      # -----------------------------------------------------------------------------
      hostname: 'twitter.com',
      port: null,
      pathname: '/search',
      search: '?p=from%3ABarackObamapetrol%20prices'
      # -----------------------------------------------------------------------------
    },
  response: undefined,
  isAxiosError: true,
  toJSON: [Function: toJSON]
}
```

### What did you expect to happen?

I was expecting the server to start and render the homepage after visiting `http://localhost:8000/`

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by TAKANOME-DEV**
I added `filterKeyword.js` in utils folder to filter all spaces in the keyword `(/src/controller/api/twitterAPIController)` and also a test file to make sure that it does work properly!

And all double quotes inside files that I opened are changed in single quote by prettier. Sorry about that :sweat: 

Close #119

### CODE REVIEW

**Comment by TAKANOME-DEV** [comment]
Seems like there is some issues with the actions. 
@MizouziE am I missing some configurations to make the deployment pass or? 
I found [this interesting issue](https://github.com/actions/upload-pages-artifact/issues/6) about the action.
Perhaps we need to add another step?
Let me know what do you think about it

**Comment by MizouziE** [comment]
@TAKANOME-DEV Ah sorry! No it's my fault cos I was messing with other options for hosting and started playing with new CI/CD stuff. It's perfectly fine to ignore at the moment as it doesn't actually deploy anywhere. I have a solution in the pipeline that will be able to make use of the actions, so I didn't bother removing it as I'll just need to update it once my server is in place.

We'll soon be saying bye-bye to heroku! 🙌

File: src/server.js [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+const apiTwitter = require('./routes/api/twitter');
+const rateLimit = require('express-rate-limit');
 
 function createServer() {
   const app = express();
 
   // set up rate limiter: maximum of 60 requests per minute
   const limiter = rateLimit({
     windowMs: 1 * 60 * 1000, // 1 minute
-    max: 60, // 1 requeest per second
```
Comment (MizouziE) [inline src/server.js:14]: Nice spot! 😅

**Review by MizouziE — APPROVED** [review]
👌

**Pull request merged**